tasks.py

```diff
@@ -90,6 +90,12 @@ def adminserver(port=8001):
     cmd = '{} python manage.py runserver {} --nothreading'.format(env, port)
     run(cmd, echo=True, pty=True)
 
+# TODO - refactor? It is functional. e.g. 'invoke admin_task createsuperuser'
+@task
+def admin_task(args):
```

Comment:
Yeah, it makes sense now. Since `args` is a required argument (i.e. there is no default), you can pass it positionally on the command line. Subsequent tokens will be treated as separate arguments. You might be able to workaround it by using quotes, like:

```python
inv admin.manage 'createsuperuser foo@bar.com'
```